packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -130,18 +115,23 @@ const promoBanners = ref<PromoBanner[]>([
   }
 ])
 
-const { activeUser, isGuest } = useActiveUser()
-const { triggerNotification } = useGlobalToast()
-const areQueriesLoading = useQueryLoading()
-const apollo = useApolloClient().client
+const {
+  on,
+  bind,
+  value: search
+} = useDebouncedTextInput({
+  debouncedBy: 800,
+  model: ref('')
```

Comment:
The model key is redundant here, the only point for it is to feed in some pre-existing model ref you want to use. If you're just creating a new one anyway, omit the model key altogether. Fix this everywhere